_includes/footer.html

```diff
@@ -5,6 +5,9 @@
 {% if site.theme_settings.footer_text %}
 <footer>
 	<div class="site-footer">
+	<p class="text">
+		<a rel="RAC Privacy Policy" href="https://rockarch.org/about-us/privacy-policy/" style="text-decoration:underline">RAC Privacy Policy</a>
+	</p>
```

Comment:
This seems overly complex. Can we do something like:
```suggestion
	<a href="https://rockarch.org/about-us/privacy-policy/">Privacy Policy</a>
```
If styling needs to happen it should be done via a class on the anchor tag, not as inline styles.

@HaSistrunk can correct me if I'm wrong but I believe that the `rel` attribute is not necessary if link text is present.

Reply:
I am definitely no CSS wiz, so this does look better to me. The only comment I'd have is the possible addition of underlining because the color contrast isn't super high on this site.

Reply:
You can do that by adding a CSS rule. The inline styles on the links in the footer are Bad Form and this would be a good opportunity to fix those.